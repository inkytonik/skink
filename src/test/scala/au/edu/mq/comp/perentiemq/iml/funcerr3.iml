// No arguments in call
def funcerr3(a) : (b) {
    a = f();
}
