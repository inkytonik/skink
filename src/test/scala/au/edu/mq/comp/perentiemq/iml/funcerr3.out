src/test/scala/au/edu/mq/comp/perentiemq/iml/funcerr3.iml:3:11: error: expression expected
    a = f();
          ^

